viewer/css/app.css

```diff
@@ -56,6 +56,27 @@ body {
   background-color: #fff;
 }
 
+.embed {
```

Comment:
Can you move all the embed related CSS before /** Search input **/ and add a comment to delimitate the section (/** Embed version **/) ?